ansible_collections/arista/avd/roles/eos_designs/docs/how-to/avd-to-cvaas.md

```diff
@@ -1,8 +1,7 @@
 # The purpose of this tutorial is to review the steps required in order to integrate AVD with Arista CVaaS.
 
-![Figure: 1](../../../../media/create_account.png)
-
 ## Within CVaaS navigate to:
+
 ### Settings and Tools --> Access Control --> Service Accounts --> "New Service Account"
```

Comment:
here it might also be better to not make the steps an actual heading and change the wording a bit

e.g.:

```
## Steps to create service accounts on CloudVision

1. Go to Settings and Tools --> Access Control --> Service Accounts --> click `+ New Service Account`

Reply:
Agreed